pkg/resourcemanager/webhook/highavailabilityconfig/handler.go

```diff
@@ -150,7 +156,8 @@ func (h *Handler) handleDeployment(
 	}
 
 	h.mutateNodeAffinity(
-		failureToleranceType,
+		// TODO Remove "failureToleranceType != nil" in the future
```

Comment:
```suggestion
		// TODO(ScheererJ): Remove "failureToleranceType != nil" in the future
```

Please make yourself a reminder to follow-up so that we don't forget about this :)

Reply:
Updated accordingly with the next commit.
I created a corresponding reminder for myself.